src/web/settings.py

```diff
@@ -44,17 +45,16 @@
 
 static_files_folders = [
     static_files_base_dir,
-    *[path for path in plugin_manager.static_paths],
+    *plugin_manager.static_paths,
 ]
 
 static_files_router: Router = create_static_files_router(
     path='/static',
-    directories=static_files_folders,
+    directories=list(static_files_folders),
```

Comment:
that's a list already, why is this cast needed?

Reply:
I'll let you try to figure that one out, but it's unhappy without it